src/pages/home/report/ReportActionsList.js

```diff
@@ -109,7 +109,7 @@ class ReportActionsList extends React.Component {
      * @return {String}
      */
     keyExtractor(item) {
-        return `${item.action.clientID}${item.action.reportActionID}${item.action.sequenceNumber}`;
+        return `${item.action.clientID}${item.action.reportActionID}`;
```

Comment:
Do we still need the `clientID` if we are switching to `reportActionID` and there will only ever be one?

Reply:
Good call, we don't need clientID because we're assuming that reportActionID is random enough to be universally unique